src/clients/aws_client_factory.py

```diff
@@ -18,6 +18,7 @@
 from src.clients.aws_organizations_client import AwsOrganizationsClient
 from src.clients.aws_ssm_client import AwsSSMClient
 from src.clients.aws_s3_client import AwsS3Client
+from src.clients.aws_hostedZones_client import AwsHostedZonesClient
```

Comment:
To stay consistent, this should be broken into aws_hosted_zones_client.

Reply:
renamed